package.json

```diff
@@ -146,5 +146,8 @@
       "dependencies",
       "blog-post"
     ]
+  },
+  "dependencies": {
+    "https-proxy-agent": "^2.2.2"
```

Comment:
Don't add dependencies to the root `package.json`. It's never published or depended on anywhere outside of the monorepo. Just having the dependency in the `package.json` of the module(s) that uses it (in this case `core`) is sufficient.

Reply:
removed it from root package.json